upgrade/upgrade-scripts/common.sh

```diff
@@ -204,3 +204,10 @@ function set_upgrade_property() {
 	source_upgrade_properties ||
 		die "failed to read properties file after setting '$1=$2'"
 }
+
+function turn_verbose_logging_on() {
+    exec 4> >(logger -t $0)
+    BASH_XTRACEFD="4"
+    PS4=':${BASH_SOURCE}:${LINENO}+'
```

Comment:
When I execute an upgrade using the "latest" version, like this:
```
$ sudo /var/dlpx-update/latest/upgrade -nv deferred
```
the the path using "latest" winds up in the log as well:
```
Nov 20 23:26:45 ps-trunk delphix-upgrade[22530]: /var/dlpx-update/latest/upgrade:103 case "$UPGRADE_TYPE" in
Nov 20 23:26:45 ps-trunk delphix-upgrade[22530]: /var/dlpx-update/latest/upgrade:111 CONTAINER=delphix.Oo4cOFI
Nov 20 23:26:45 ps-trunk delphix-upgrade[22530]: /var/dlpx-update/latest/upgrade:112 [[ -n delphix.Oo4cOFI ]]
```
Ideally, I think we'd want to show the "real" path, with the upgrade image version embedded in it, rather than "latest".

It's probably not worth the effort to fix this, but I at least wanted to mention it.